components/dashboard-modules/EnergyRecommendation.tsx

```diff
@@ -38,7 +39,24 @@ const EnergyRecommendation: FC<{ siteUUID: string }> = ({ siteUUID }) => {
     : null;
 
   const { isDayTime } = useTime(latitude, longitude);
-  if (!isDayTime && currentOutput === 0) {
+  if (isLoading) {
+    return (
+      <div
+        className="
+        flex-1
+        flex
+        p-4
+        text-center
+        justify-center
+        align-center
+        bg-ocf-black-500
+        rounded-2xl
+        h-[100%]"
+      >
+        <div className="bg-ocf-gray-1000 w-[100%] rounded-3xl animate-pulse"></div>
+      </div>
+    );
+  } else if (!isDayTime && currentOutput === 0) {
```

Comment:
The `<EnergyRecommendation` component on the dashboard doesn't fade its skeleton layout out similar to the `<SunnyTimeframe` module (read other comment). These lines are the reason in this case, though I'm not sure what the best fixing option is for this one.

For now, let's leave it be and create an issue to track it.